src/pages/index.tsx

```diff
@@ -10,6 +10,17 @@ const Home: NextPage = () => {
   const { data } = api.repos.getRepos.useQuery();
   const [repos, setRepos] = useState(data);
   const [filter, setFilter] = useState<Set<string>>(new Set());
+  const [isSearching, setIsSearching] = useState<string>("");
```

Comment:
`isSearching` makes me believe it's a boolean value but it's really a search string. The name should reflect the intent... maybe `searchVal` 🤷